src/lib/y2packager/repository.rb

```diff
@@ -117,12 +123,14 @@ def create(name:, url:, enabled: true, autorefresh: true)
     # @param enabled     [Boolean]      Is the repository enabled?
     # @param autorefresh [Boolean]      Is auto-refresh enabled for this repository?
     # @param url         [URI::Generic] Repository URL
-    def initialize(repo_id:, name:, enabled:, autorefresh:, url:)
+    # @param product_dir [String]       Product directory
+    def initialize(repo_id:, name:, enabled:, autorefresh:, url:, product_dir: "/")
```

Comment:
Um, the `create` method uses an empty string as the default for `product_dir`, any reason for this inconsistency?